packages/stores/src/providers/P2PSettingsProvider.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { ReactNode } from 'react';
+import { useSubscription } from '@deriv/api';
+import P2PSettingsContext, { TP2PSettings } from '../stores/P2PSettingsContext';
+import { useLocalStorage } from 'usehooks-ts';
+
+type TP2PSettingsProvider = {
+    children: ReactNode;
+};
+
+const P2PSettingsProvider = ({ children }: TP2PSettingsProvider) => {
+    const { data, subscribe, ...rest } = useSubscription('p2p_settings');
+    const [p2p_settings, setP2PSettings] = useLocalStorage<TP2PSettings>('p2p_settings', {} as TP2PSettings);
+
+    React.useEffect(() => {
+        if (data) {
```

Comment:
```suggestion
        if (!data) return
```

Reply:
isn't it already enough that I have the check below for p2p_settings? 

if (!p2p_settings_data) return undefined;